doc/nrf/ug_thread.rst

```diff
@@ -16,10 +16,15 @@ For information about how to start working with Thread in |NCS|, see the followi
 
 See :ref:`openthread_samples` for the list of available Thread samples.
 
+Thread tools
+************
+
```

Comment:
Small rebase issue here. These lines should be removed, as they have been removed on master.

Reply:
Done